JobConfig/digitize/DTSfilter.fcl

```diff
@@ -0,0 +1,38 @@
+#
+#  to filter DTS
+#
+#include "Production/JobConfig/common/prolog.fcl"
+#include "Production/JobConfig/digitize/prolog.fcl"
+# name all processes the same, to mask any provenance in ensemble creation
+process_name: Digitize
+source : { module_type : RootInput }
+services : @local::Services.SimAndReco 
+physics : {
+  producers : {
+    @table::Digitize.producers
+    @table::Trigger.producers
+  }
+  filters: {
+    @table::Digitize.filters
+    @table::Trigger.filters
+  }
+  analyzers: { @table::Digitize.analyzers }
+  # define the digitization paths.  Trigger paths are added specific to digitization type
+  DigitizePath : @local::Digitize.DigitizeSequence
```

Comment:
We want to slightly loosen the normal digi filter here, as we are going to re-digitize these events later and want to have a buffer.  ie subtract 1 or 2 from the # of required hits.

Reply:
Sure, I need to understand what filters are called here